app/components/ui/atoms/ManuscriptStatus.js

```diff
@@ -6,12 +6,14 @@ import PropTypes from 'prop-types'
 const StatusBox = styled(Box)`
   color: ${props => props.theme[props.color]};
   font-size: 16px;
+  padding-left: 24px;
+  flex: 0 0 120px;
```

Comment:
FYI we try to use values from the theme whenever possible. In this case `fontSizeBase`, `space.3` and `space.7`.